docs/commands/login.md

```diff
@@ -14,6 +14,22 @@ Also, you will be able to access your members-only (private) API definitions in
 
 If you're having issues with the `login` command, use the `--verbose` option to display a detailed error trace (if any).
 
+## Region
```

Comment:
@skadinna I think we should reshape so it doesn't feel like users can choose where to login. It depends on the region their org is in. It can be only one region per org.

I would change it to something like:

if your redocly organization is in EU region then ...